pages/pipelines/trigger_step.md.erb

```diff
@@ -9,6 +9,18 @@ steps:
   - trigger: "another-pipeline"
 ```
 
+All builds created by a trigger step will be authored by the same user that created the initial build. This user must:
```

Comment:
“will be authored by” still feels very awkward to me

Reply:
"All builds created by a trigger step will be created by the user that triggered the original build, or the user that unblocked the last preceding block step. That user must be:"

Reply:
Possibly with "If the build was triggered by a commit, then the user will be inferred from the commit author"?